jobserver/views/projects.py

```diff
@@ -173,7 +173,7 @@ def get_repo(repo, ctx):
                     is_private = self.get_github_api().get_repo_is_private(
                         repo.owner, repo.name
                     )
-                except requests.HTTPError:
+                except (requests.HTTPError, TimeoutError):
```

Comment:
I'm curious as to why `TimeoutError` and not `requests.Timeout`.

Reply:
I wasn't aware of the latter, and typed the former out of habit! I think it'll be better with `requests.Timeout` having read the docs, thanks!

Reply:
I don't think this has been resolved in the latest commits, unless I'm missing something.

I think it's important to use requests.Timeout here not TimeoutError. I don't yet have access to sentry so I can't check the report -- but I assume it was requests.TimeoutError that would be raised if the GH API did indeed time-out. So I don't think the issue would be resolved without this change.

The [Python docs](https://docs.python.org/3/library/exceptions.html) say the builtin TimeoutError is something different, nothing to do with HTTP in general, I think:

> Raised when a system function timed out at the system level. Corresponds to errno [ETIMEDOUT](https://docs.python.org/3/library/errno.html#errno.ETIMEDOUT).

[requests.Timeout](https://github.com/psf/requests/blob/1ae6fc3137a11e11565ed22436aa1e77277ac98c/src/requests/exceptions.py#L71) does not sub-class TimeoutError.

Reply:
>but I assume it was requests.TimeoutError

`requests` doesn't have `TimeoutError`, only `requests.Timeout`. The sentry issue is definitely the former